app/Resources/AppBundle/assets/js/main.js

```diff
@@ -2,11 +2,11 @@ require('./MovingCards');
 
 // materialize-css element functions
 $(document).ready(function(){
+    $(".dropdown-button").dropdown();
```

Comment:
It's all for main menu, isn't it? What about putting related stuff into a file called ```menu.js``` or something like that?